justgage.js

```diff
@@ -724,7 +724,7 @@
 
     // set value to display
     if (obj.config.textRenderer) {
-      obj.originalValue = obj.config.textRenderer(obj.originalValue);
+      obj.originalValue = obj.config.textRenderer(obj.originalValue, obj.config);
```

Comment:
ATM this is the main justgage file, the other one is a copy of this and is used on docs. You have something missing in both files. For example in this you are missing the return value check and in the other you are missing the config passed to the textrender function

Reply:
Sorry - easy to overlook